packages/go_router/lib/src/router.dart

```diff
@@ -132,8 +132,6 @@ class GoRouter extends ChangeNotifier implements RouterConfig<RouteMatchList> {
       _routeInformationParser;
 
   /// The route configuration. Used for testing.
-  // TODO(johnpryan): Remove this, integration tests shouldn't need access
-  @visibleForTesting
   RouteConfiguration get routeConfiguration => _routeConfiguration;
```

Comment:
We can keep this private if the defaultBranchLocation doesn't rely on routeConfiguration